discovery-infra/test_infra/utils.py

```diff
@@ -555,6 +555,12 @@ def create_ip_address_list(node_count, starting_ip_addr):
     return [str(ipaddress.ip_address(starting_ip_addr) + i) for i in range(node_count)]
 
 
+def create_ip_address_nested_list(node_count, starting_ip_addr):
+    return [[str(ipaddress.ip_address(starting_ip_addr) + i)] for i in range(node_count)]
+
+def create_empty_nested_list(node_count):
```

Comment:
don't see any usage of it

Reply:
https://github.com/empovit/assisted-test-infra/blob/93c536cfe495442ce6aff0779f8f5767e9a0a143/discovery-infra%2Fstart_discovery.py#L84
https://github.com/empovit/assisted-test-infra/blob/93c536cfe495442ce6aff0779f8f5767e9a0a143/discovery-infra%2Fstart_discovery.py#L155